docs/running/running.rst

```diff
@@ -1,20 +1,18 @@
-.. highlight:: console
-
 .. _running:
 
 Running Toil workflows
 ======================
 
-.. _quickstart:
-
-Quickstart: A simple workflow
------------------------------
+This section describes how to run Toil workflows.
 
-Starting with Python, a Toil workflow can be run with just three steps.
+.. _quickstart:
 
-1. Install Toil (see :ref:`installation-ref`)::
+Toil Quickstart
+---------------
 
-      $ pip install toil
+A Toil workflow can be run with just three steps.
+ 
```

Comment:
I think we should mention virtualenv here. If they don't do it, they are gonna have a bad time.